### DISCUSSION TITLE
中文讨论须知

### DISCUSSION THREAD

**Discussion by unknwon** [gogs/gogs#6665] [中文讨论]
感谢您使用 Gogs 并愿意参与讨论，您可以在该子板块使用中文进行常规讨论，但该板块不接受问题反馈和功能提需。

如需进行问题反馈和功能提需，请使用我们预设的[问题模板](https://github.com/gogs/gogs/issues/new/choose)并使用英文完成提交。

可加入 “Gogs 技术交流群” QQ 群 218443817。

**Comment by QASIMARA** [comment]
這。。。很有意思。

**Comment by manongqiyin** [comment]
用docker-compose运行的报错usermod: no changes是为啥，咋能解决

**Comment by QASIMARA** [comment]
what?


Sincerely,


David E. Schnell

________________________________
